superset-frontend/packages/superset-ui-core/src/query/getAxis.ts

```diff
@@ -16,20 +16,18 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import {
-  DTTM_ALIAS,
-  getColumnLabel,
-  isDefined,
-  QueryFormData,
-} from '@superset-ui/core';
+import { DTTM_ALIAS, getColumnLabel, QueryFormData } from '@superset-ui/core';
+
+export const isEnabledAxes = (formData: QueryFormData) => !!formData.x_axis;
```

Comment:
Could we rename this to `isXAxisEnabled` or better yet `isXAxisSet` for clarity?

Reply:
Make sense, thanks.